src/Client/CMakeLists.txt

```
@@ -27,6 +27,7 @@ else()
 endif()
 
 target_link_libraries(${CURRENT_PROJECT} PUBLIC ${PROJECT_NAME}Core TestLvl)
+target_link_libraries(${CURRENT_PROJECT} PUBLIC ${PROJECT_NAME}Core AbstractLvl)
```

```suggestion
target_link_libraries(${CURRENT_PROJECT} PUBLIC ${PROJECT_NAME}Core TestLvl AbstractLvl)
```

use on line for include libraries